features/admins/users/admin_manage_users.feature

```diff
@@ -60,13 +60,15 @@ Feature: Admin Actions to manage users
     Given the user "new_user" exists and is activated
       And I am logged in as an admin
     When I go to the abuse administration page for "new_user"
-    Then I should see "No login recorded"
+    Then I should see "Current Login No login recorded"
+      And I should see "Previous Login No previous login recorded"
     When time is frozen at 1/1/2019
       And I am logged in as "new_user"
+      And I am logged out
       And I jump in our Delorean and return to the present
       And I am logged in as an admin
       And I go to the abuse administration page for "new_user"
     Then I should not see "No login recorded"
-      And I should see "2019-01-01 12:00:00 -0500 (127.0.0.1) Current Login System Generated"
-      And I should see "2019-01-01 12:00:00 -0500 (127.0.0.1) Previous Login System Generated"
+      And I should see "Current Login IP Address: 127.0.0.1 2019-01-01 12:00:00 -0500"
```

Comment:
The timestamp will be in the first column now, so I think this is going to fail.

Reply:
Ohh, I know why this is passing: it's reading the timestamp for the Previous Login, which is on the next line and, due to the time freezing, happens to be the same as the one for the Current Login. The steps should be updated so the timestamp is the first thing, i.e. `And I should see "2019-01-01 12:00:00 -0500 Current Login IP Address: 127.0.0.1"` and `And I should see "2019-01-01 12:00:00 -0500 Previous Login IP Address: 127.0.0.1"`